Package.swift

```diff
@@ -0,0 +1,223 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import PackageDescription
+
+let package = Package(
+  name: "GoogleUtilities",
+  platforms: [.iOS(.v10), .macOS(.v10_12), .tvOS(.v10), .watchOS(.v6)],
+  products: [
+    .library(
+      name: "AppDelegateSwizzler",
+      targets: ["GoogleUtilities_AppDelegateSwizzler"]
+    ),
+    .library(
+      name: "UserDefaults",
+      targets: ["GoogleUtilities_UserDefaults"]
+    ),
+  ],
+  dependencies: [
+    .package(name: "Promises", url: "https://github.com/google/promises.git", "1.2.8" ..< "1.3.0"),
+    .package(
+      name: "OCMock",
+      url: "https://github.com/firebase/ocmock.git",
+      .revision("7291762d3551c5c7e31c49cce40a0e391a52e889")
+    ),
+  ],
+  targets: [
+    .target(
+      name: "GoogleUtilities_AppDelegateSwizzler",
+      dependencies: ["GoogleUtilities_Environment",
+                     "GoogleUtilities_Logger",
+                     "GoogleUtilities_Network"],
+      path: "GoogleUtilities",
+      exclude: [
+        "CHANGELOG.md",
+        "CMakeLists.txt",
+        "LICENSE",
+        "README.md",
+        "AppDelegateSwizzler/README.md",
+        "Environment/",
```

Comment:
Excluding path manually looks not ideal :( I wonder if we can apply some changes to the folder structure to avoid this?